Kiosk/CognitiveServiceApiKeyTester.cs

```diff
@@ -154,6 +157,24 @@ private static async Task<HttpResponseMessage> RequestAndAutoRetryWhenThrottled(
 
             return response;
         }
+
+        public static async Task TestInkRecognizerApiKeyAsync(string key, string apiEndpoint)
+        {
+            bool isUri = !string.IsNullOrEmpty(apiEndpoint) ? Uri.IsWellFormedUriString(apiEndpoint, UriKind.Absolute) : false;
+            if (!isUri)
+            {
+                throw new ArgumentException("Invalid URI");
+            }
+            else
+            {
+                string inkRecognitionUrl = "/inkrecognizer/v1.0-preview/recognize";
```

Comment:
>/inkrecognizer/v1.0-preview/recogniz [](start = 44, length = 36)

Hmm these should be constants but looks like eveyrthing else in this file is hard coded....

Do you have this value anywhere else?

Reply:
This is the only place that this value is used within this file. I don't see any harm in making the URL string a constant, but I am not sure if it is necessary or not.